README.md

```diff
@@ -32,10 +33,23 @@ tests/test_coverage.py::test_coverage PASSED                             [100%]
 =========================== 6 passed in 7.08 seconds ===========================
 ```
 
+### Test Profiles
+
+The integration tests support two test profiles:
+- **devel**: this is the recommended profile for running the integration tests on
+  a local development machine.
+- **ci** (default option): this is the profile used when running the integration tests as part of
+  the the Continuous Integration (CI).
+
```

Comment:
Could we describe the main differences between the 2 profiles?

Reply:
For now the only difference is in running the coverage test. This difference is described in the Adaptive Coverage section of the doc. For this particular crate I don't believe we will need anything else, but for the virtio-devices we might want to do performance testing and we probably cannot expect to have the same thresholds on a local machine vs on a i3.metal. We can use the same model for these crates in the future.

I have no good idea about describing the difference between the profiles. Should I just say that the coverage test behaves differently with different profiles? Do you have any better suggestions?

Reply:
Done.